webapp/app.py

```diff
@@ -605,13 +607,18 @@ def takeovers_index():
 
 @app.after_request
 def cache_headers(response):
-    """
-    Set cache expiry to 60 seconds for homepage and blog page
-    """
+    # Set cache to private to prevent caching
 
     if flask.request.path in ["/core/build"]:
         response.cache_control.private = True
 
+    if flask.request.path in ["/account.json"]:
+        response.cache_control.private = True
+        response.headers[
+            "Cache-Control"
+        ] = "no-cache, no-store, must-revalidate"
```

Comment:
Rather than chucking everything at the problem, let's work out what we actually need. If you have no-store, you shouldn't need no-cache as well. In fact I think there's a chance that no-cache might override no-store in some cases.

I don't know if there's any relevant client that would need the `pragma` header as well as the `cache-control` one. I think probably not.

I don't know how both interact with must-revalidate.

Then there's the question of whether we actually want to prevent caching completely here. We probably do, but we could consider allowing the user's browser to cache the response for a bit. I'll think and read about it a bit.